mlir/include/Gradient/IR/GradientOps.td

```diff
@@ -119,6 +120,51 @@ def BackpropOp : Gradient_Op<"backprop", [AttrSizedOperandSegments]> {
 }
 
 
+def JVPOp : Gradient_Op<"jvp", [DeclareOpInterfaceMethods<CallOpInterface>,
+        DeclareOpInterfaceMethods<SymbolUserOpInterface>]> {
+    let summary = "Compute the jvp of a function.";
+
+    let arguments = (ins
+        StrAttr:$method,
+        FlatSymbolRefAttr:$callee,
+        Variadic<AnyType>:$params_and_tangents,
```

Comment:
Agree, it makes sense. Initially I didn't find how to describe the two lists of parameters in TableGen, and decided to go further to solve some more important parts of the PR. I will look once again into it soon.

Reply:
Ah, yes: If I declare two subsequent Variadic arrays in TableGen I got  `error: op has multiple variadic operands but no specification over their sizes`. it is not obvious to me how to link the number of callee's symbolic arguments to the size of Variadic array in TableGen so I left this problem for later.

Reply:
When all variadic operands have the size, the `SameVariadicOperandSize` trait can be attached to indicate this, otherwise the `AttrSizedOperendsSegments` trait is required. See here: https://mlir.llvm.org/docs/DefiningDialects/Operations/#variadic-operands

Reply:
It should be done now, thanks for the suggestions!